runner/src/server/plugins/engine/Auth.ts

```diff
@@ -51,7 +51,7 @@ export default {
     plugin: {
         name: "auth",
         register: async (server: HapiServer) => {
-            if (config.jwtAuthEnabled) {
+            if (config.jwtAuthEnabled && config.jwtAuthEnabled !== "false") {
```

Comment:
A very minor question for this and the other times this is done in the PR, if the value for this will be `"true"` it might be better do to an equality check for that specifically: 

```js
if (config.jwtAuthEnabled === "true") {
}
```

The (very unlikely) concern might be that someone accidentally makes the value "[truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)" (i.e defined) but didn't intend for it to be enabled by explicitly setting `"true"`.

Checking it equals true might be a bit simpler to read too.

(definitely doesn't feel like something that needs to hold up the PR!)

Reply:
better to have true